faqs/galaxy/purge histories

```diff
@@ -0,0 +1,15 @@
+---
+title: Find all Histories and purge (aka permanently delete)
+area: Account      
+box_type: question
```

Comment:
Why did you choose `box_type: question`? I would have chosen `tip`

Reply:
ok thanks updated to tip